keps/sig-cloud-provider/providers/2020-cloud-provider-inspur.md

```diff
@@ -0,0 +1,145 @@
+and enables them to take better advantages of potential business opportunities. 
+It also helps small- and medium-sized enterprises expand their business growth and rise to challenges. 
+
+The [Inspur Open Platform (IOP)](https://cloud.inspur.com/product/UK8S/) is an application-oriented container management platform based on arm64 architecture. It provides complete Kubernetes clustering capabilities, helping users to quickly deploy flexible and reliable container clusters, easily create and manage container workloads, and provide automatic scaling and resource monitoring. Efficient operation and maintenance capabilities such as log collection and retrieval. 
+IOP has been using this project for several releases:
+- v1.15 [Conformance Test Report](https://github.com/cncf/k8s-conformance/tree/843ee84d40962baa07cab9e59a19abe7f778b6b0/v1.15/INSPUR-IOP)
+- v1.13 [Conformance Test Report](https://github.com/cncf/k8s-conformance/tree/843ee84d40962baa07cab9e59a19abe7f778b6b0/v1.13/INSPUR-IOP)
+- v1.11 [Conformance Test Report](https://github.com/cncf/k8s-conformance/tree/843ee84d40962baa07cab9e59a19abe7f778b6b0/v1.11/INSPUR-IOP)
+- v1.9 [Conformance Test Report](https://github.com/cncf/k8s-conformance/tree/843ee84d40962baa07cab9e59a19abe7f778b6b0/v1.9/INSPUR-IOP)
+
+As well as IOP, another certified Kubernetes product which used this project is `INSPUR IOP`, and you can refer to 
+following conformance test reports:
+- v1.17 [Conformance Test Report for amd64](https://github.com/cncf/k8s-conformance/blob/master/v1.17/inspur-iop-amd64)
+- v1.17 [Conformance Test Report for arm64](https://github.com/cncf/k8s-conformance/blob/master/v1.17/inspur-iop-arm64)
+- v1.16 [Conformance Test Report for amd64](https://github.com/cncf/k8s-conformance/blob/master/v1.16/inspur-iop-amd64)
+- v1.16 [Conformance Test Report for arm64](https://github.com/cncf/k8s-conformance/blob/master/v1.16/inspur-iop-arm64)
+- v1.15 [Conformance Test Report for amd64](https://github.com/cncf/k8s-conformance/blob/master/v1.15/inspur-iop-amd64)
+- v1.15 [Conformance Test Report for arm64](https://github.com/cncf/k8s-conformance/blob/master/v1.15/inspur-iop-arm64)
+- v1.14 [Conformance Test Report for amd64](https://github.com/cncf/k8s-conformance/blob/master/v1.14/inspur-iop-amd64)
+- v1.14 [Conformance Test Report for arm64](https://github.com/cncf/k8s-conformance/blob/master/v1.14/inspur-iop-arm64)
+
+Other usage of this project can be seen from [GitHub issues](https://github.com/OpenInspur/cloud-provider-inspur/issues).
+
+### Testgrid Integration
+
+INSPUR cloud provider is reporting conformance test results to TestGrid as per the [Reporting Conformance Test Results to Testgrid KEP](https://github.com/kubernetes/enhancements/blob/6427a0becff459815e0e41f72f65ab5f3b8e9c6d/keps/sig-cloud-provider/0018-testgrid-conformance-e2e.md).
+You can get the result from [TestGrid INSPUR Cloud Dashboard](https://testgrid.k8s.io/conformance-cloud-provider-INSPURcloud). 
```

Comment:
There does not appear to be a corresponding dashboard.

Reply:
@cheftako  I have requested the GCS bucket and the [PR](https://github.com/kubernetes/k8s.io/pull/1426) got merged, but I'm not sure what should I do next, will that dashboard be created by CI automatically or by @dim who approved the PR?